site/content/docs/5.1/utilities/api.md

```diff
@@ -15,8 +15,8 @@ The `$utilities` map contains all our utilities and is later merged with your cu
 | Option | Type | Default&nbsp;value | Description |
 | --- | --- | --- | --- |
 | [`property`](#property) | **Required** | – | Name of the property, this can be a string or an array of strings (e.g., horizontal paddings or margins). |
-| [`values`](#values) | **Required** | – | List of values, or a map if you don't want the class name to be the same as the value. If `null` is used as map key, it isn't compiled. |
-| [`class`](#class) | Optional | null | Name of the generated class. If not provided and `property` is an array of strings, `class` will default to the first element of the `property` array. |
+| [`values`](#values) | **Required** | – | List of values, or a map if you don't want the class name to be the same as the value. If `null` is used as map key, `class` is not prepended to the class name. |
+| [`class`](#class) | Optional | null | Name of the generated class. If not provided and `property` is an array of strings, `class` will default to the first element of the `property` array. If not provided and `property` is string, class names are `values` keys. |
```

Comment:
> If not provided and `property` is string, class names are `values` keys.

the language here needs tweaking, but i'm not quite sure what that is supposed to mean. Taking a guess, is it something along these lines? "If not provided and `property` is a string, the `values` keys are used for the `class` names."

Reply:
reading further down, it seems i'm correct with my assumption. will make a further tweak here...

Reply:
https://github.com/twbs/bootstrap/blob/33c7f9fe6940f2972928174177ef834529b7a400/scss/mixins/_utilities.scss#L28

Reply:
https://github.com/twbs/bootstrap/blob/33c7f9fe6940f2972928174177ef834529b7a400/scss/_utilities.scss#L613